about.html

```diff
@@ -0,0 +1,127 @@
+                </p>
+                <p class="about-last">Please contact us per Email for any further questions about AE Global Summit 2023! <br> <b class="about-mail">aesummit2023@aenig.org</b></p>
+                
+            </div>
+       </div>
+    </section>
+    <section class="second-about-section">
+        <h4 class="about-second-header">Agricultural Enthusiast Global Summit 2023 Logo<hr class="horizontal-line"></h4>
+        <p class="about-logo">
+            The logo of AE Global Summit 2023 was decided through the tech-agro competition from 14. February to 16. March<span class="desk-addition foot-section">The winning designer, Adebola Adebayo, was thrilled to receive the prize money and recognition for their work. The logo was immediately implemented on the summit's website, packaging, and social media platforms, and it has been received positively by delegates.</span>
+        </p>
+        <div class="logo-inside">
+            <img class="about-into-view" src="images/my-logo.png" alt="logo">
+        </div>
+    </section>
+    <section class="third-about-section">
+        <h4 class="about-second-header">Beyond AE Global Summits<hr class="horizontal-line"></h4>
+        <p class="about-second-header-text">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. In eum aliquam, harum aperiam exercitationem accusantium assumenda blanditiis, quis accusamus ad, ex a eveniet similique rem dolores voluptatem suscipit sapiente consectetur?
+        </p>
+        <div class="desk-about-sum-display">
+            <div class="bg-div-1 item-sum-1">
+                <h2 class="inside-text-date">2020</h2>
+                <p class="sum-loc">AE Global Summit 2023 in Nairobi</p>
+            </div>
+            <div class="bg-div-1 item-sum-1">
+                <h2 class="inside-text-date">2021</h2>
+                <p class="sum-loc">AE Global Summit 2023 in Kampala</p>
+            </div>
+        </div>
+        <div class="bg-div-1 about-sum-ind">
+            <h2 class="inside-text-date">2020</h2>
+            <p class="sum-loc">AE Global Summit 2023 in Nairobi</p>
+        </div>
+        <div class="bg-div-1 about-sum-ind">
+            <h2 class="inside-text-date">2021</h2>
+            <p class="sum-loc">AE Global Summit 2023 in Kampala</p>
+        </div>
```

Comment:
- [ ] This part is supposed to have two images with text centered over them but here I see only the text. The image is a part of HTML and not a part of CSS, you are not supposed to use it has background image but as an image in the HTML file the you style the part in the CSS file, kindly fix that 🙃